changelog.dd

```diff
@@ -8,11 +8,26 @@ $(BUGSTITLE Library Changes,
 
 $(LI $(REF GCAllocator.goodAllocSize, std,experimental,allocator,gc_allocator)
 was added.)
+$(LI $(RELATIVE_LINK2 std-algorithm-iteration-scan, $(XREF algorithm.iterator,
+scan) was added.))
 
 )
 
 $(BUGSTITLE Library Changes,
 
+$(LI $(LNAME2 std-algorithm-iteration-scan, $(XREF algorithm.iterator, scan)
```

Comment:
`XREF` doesn't work like this. Please use `$(REF scan, std, algorithm, iteration)`